src/templates/QuickstartDetails.js

```diff
@@ -109,17 +109,40 @@ const QuickstartDetails = ({ data, location }) => {
             css={css`
               border-bottom: none;
               display: grid;
-              padding-bottom: 0;
-              grid-template-areas: 'title logo' 'summ logo' 'cta logo';
               grid-column-gap: 1rem;
               grid-row-gap: 1rem;
+              grid-template-areas:
+                'title logo'
+                'summ logo'
+                'cta logo';
+              justify-content: normal;
+              justify-self: center;
               row-gap: 1rem;
+              width: 101%;
 
               h1 {
                 font-weight: normal;
                 grid-area: title;
                 padding-bottom: 1rem;
               }
+
+              @media (min-width: 760px) {
+                background: var(--primary-background-color);
+                border: 1px solid var(--border-color);
+                border-radius: 0.25rem;
+                box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+                grid-template-areas:
+                  'logo title cta'
+                  'logo summ cta';
+                padding: 8px 0;
+                position: sticky;
+                top: var(--global-header-height);
+                z-index: 80;
+              }
+
+              .dark-mode & {
```

Comment:
I think you can do without this `&`, we're just trying to target the `dark-mode` class here 👍